api/src/main/java/marquez/db/migrations/V44_2__BackfillAirflowParentRuns.java

```diff
@@ -76,8 +76,8 @@ INSERT INTO jobs (uuid, type, created_at, updated_at, namespace_uuid, name, desc
       current_location
       FROM jobs
       WHERE namespace_name=:namespace AND name=:jobName
-      ON CONFLICT(name, namespace_uuid) WHERE parent_job_uuid IS NULL
-      DO UPDATE SET updated_at=now()
+      ON CONFLICT (name, namespace_uuid) WHERE parent_job_uuid IS NULL
+      DO UPDATE SET updated_at=EXCLUDED.updated_at
```

Comment:
I would assume the `updated_at` column would set to `now()` only when the metadata has changed. Do we want always to use default to `EXCLUDED.updated_at` even when metadata has changed?

Reply:
If we don't update the record, it won't return anything :(

Reply:
Very very sad....